word count reset

harish-kumar13 opened the pull request:
changed onkeypress to onkeyup event

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here by typing #link-number e.g. resolves #1. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.

harish-kumar13 committed:
word count reset

changed onkeypress to onkeyup event

harish-kumar13 commented:
this is my first small contribution to github. just changed the keypress event to keyup event which will trap all keystrokes

jemimaabu commented on .gitignore:
```diff
@@ -0,0 +1,2810 @@
+
```
A .gitignore file is not needed so you can delete this

harish-kumar13 replied:
done

harish-kumar13 replied:
fixed the bug where white space was being counted as character

harish-kumar13 replied:
fixed the most frequent words casing problem

harish-kumar13 committed:
deleted gitignore

deleted gitignore as requested

harish-kumar13 committed:
white space in character count

fixed the bug where white space is being counted as character

harish-kumar13 committed:
most frequent word casing issue

most frequent word casing issue is fixed where hello and Hello were not counted as 2
